### ISSUE TITLE
Introduce ST-20 token

### ISSUE THREAD

**Issue by dougiebuckets** [OpenZeppelin/openzeppelin-contracts#1215]
## 🎉 Description

It's probably too early for this feature request, but at least we'll have it logged. Once the EIP is approved, will it make sense for us to introduce the new Security Token (ST-20) to OZ? It's a backwards compatible extension of ERC-20 being used to issue restricted securities tokens in a regulated way.

Here's what the interface looks like:

```
contract IST20 { 
  // off-chain hash 
  bytes32 public tokenDetails; 

  //transfer, transferFrom must respect the result of verifyTransfer
  function verifyTransfer(address _from, address _to, uint256 _amount) view public returns (bool success);
  //used to create tokens 
  function mint(address _investor, uint256 _amount) public returns (bool success); 
}
```
Here's additional info re: ST-20: https://thesecuritytokenstandard.org/

- [x] 📈 This is a feature request.

**Comment by dougiebuckets** [comment]
According to Polymath, a draft of the EIP will be submitted end of Aug. https://twitter.com/PolymathNetwork/status/1030187810916061184

**Comment by andreafspeziale** [comment]
Hi there!
It's more than a month that I'm trying to guide myself in the vastly world of the security tokens.
I think this topic will always be more central.
I would like to share right here some links that could help:

- https://eips.ethereum.org/EIPS/eip-1462
- https://github.com/ethereum/EIPs/issues/1411

@dougiebuckets Polymath is on the front line since the topic is their core business.
The proposal is very detailed and fragmented.

So for my understanding and for what I'm seeing out there everyone is more or less applying their own rules and patterns. I think would be difficult to propose some base contracts in the OZ library (even an interface) but would be great to have OZ onboard in this topic.

For my understanding it seems also that one of the biggest pain point is (at the hight level) a sharable KYC system.
Since every issuer will use not only its contracts but its KYC approach.

Thanks guys and sorry my 2 cents!

**Comment by nventuro** [comment]
@andreafspeziale so your conclusion is that it is still to early for the community to settle on a standard implementation? You mentioned 'it'd be great to have OZ onboard', are you referring to the whole security token discussion, or to a standard in particular?

Thanks!

**Comment by Amxx** [comment]
It appears that this is still not standardized, and doesn't appear to be relevant/requested anymore.

We are reluctant to implement any new token standard that has not already been accepted by the community (ideally through a last-call or final ERC and some adoption). If this becomes relevant again, please open a new issue.
